tests/ifconfig_tests.py

```diff
@@ -98,6 +98,8 @@ def test_macosx(self):
         eq_(interfaces['en0']['inet'], '192.168.0.1')
         eq_(interfaces['en0']['broadcast'], '192.168.0.255')
         eq_(interfaces['en0']['netmask'], '255.255.255.0')
+        eq_(interfaces['lo0']['netmasks'], ['255.0.0.0'])
+        eq_(interfaces['lo0']['prefixlens'], ['64', '128'])
```

Comment:
Could we add a check for `broadcasts` here?

Reply:
Done.